examples/using-type-definitions/src/components/seo.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Helmet from 'react-helmet'
+import { StaticQuery, graphql } from 'gatsby'
```

Comment:
better use here the new `useStaticQuery`? -> https://www.gatsbyjs.org/docs/use-static-query/

Reply:
see above